src/main/java/Ui.java

```diff
@@ -0,0 +1,9 @@
+public class Ui {
+    protected String inp;
+    public Ui() {}
+    public void printStart() {
+        System.out.println("");
+        System.out.println("Hello! I'm Neo");
+        System.out.println("What can I do for you?");
+    }
+}
```

Comment:
The Ui (User interface) class is supposed to handle all the interactions with the user, suggestion for you to add all the `System.out.println(...)` code found in all the *Commands.java classes into this class as methods for Ui to handle everything.